### ISSUE TITLE
Makefile issues

### ISSUE THREAD

**Issue by t-b** [byte-physics/igor-unit-testing-framework#23]
All with clean environment via `git clean -fdx`:

```
make links # fails
make debug # fails
make docu-doxygen # fails
```

I suspect it is because for a normal build we use the docu-*.sh files and for the non-working targets we use just the makefile targets.

Can we not just drop the `docu-*.sh` files and include that logic in the Makefiles?

See https://github.com/byte-physics/igor-unit-testing-framework/blob/448d35edc618b03aba39d8aa2178bed5acadddfe/docu/docker/sphinx/Makefile#L34-L40

**Comment by t-b** [comment]
Another odd thing is the "Continue Y/N" questions as these don't work well with CI.

**Comment by t-b** [comment]
And we should switch to using a docker executioner with sphinx/doxygen images from our registry. Or even one image for all.

**Comment by t-b** [comment]
Closed via #71.
